### PR TITLE
Issue #OB-5 feat: Cloud storage sdk changes to support for GCP

### PR DESCRIPTION
**Pull request by sowmya-dixit**
Jira ticket - [OB-5](https://project-sunbird.atlassian.net/browse/OB-5)

### CODE REVIEW

**Review by maheshkumargangula — APPROVED** [review]

File: src/main/scala/org/sunbird/cloud/storage/conf/AppConf.scala [hunk 1]
```diff
@@ -35,13 +35,15 @@ object AppConf {
         if (`type`.equals("aws")) getConfig("aws_storage_key");
         else if (`type`.equals("azure")) getConfig("azure_storage_key");
         else if (`type`.equals("cephs3")) getConfig("cephs3_storage_key");
+        else if (`type`.equals("gcloud")) getConfig("gcloud_client_email");
```
Comment (maheshkumargangula) [inline src/main/scala/org/sunbird/cloud/storage/conf/AppConf.scala:38]:
```suggestion
        else if (`type`.equals("gcloud")) getConfig("gcloud_client_key");
```

File: src/main/scala/org/sunbird/cloud/storage/conf/AppConf.scala [hunk 2]
```diff
@@ -35,13 +35,15 @@ object AppConf {
...
         else "";
     }
 
     def getStorageSecret(`type`: String): String = {
         if (`type`.equals("aws")) getConfig("aws_storage_secret");
         else if (`type`.equals("azure")) getConfig("azure_storage_secret");
         else if (`type`.equals("cephs3")) getConfig("cephs3_storage_secret");
+        else if (`type`.equals("gcloud")) getConfig("gcloud_private_key");
```
Comment (maheshkumargangula) [inline src/main/scala/org/sunbird/cloud/storage/conf/AppConf.scala:46]:
```suggestion
        else if (`type`.equals("gcloud")) getConfig("gcloud_private_secret");
```

File: src/main/scala/org/sunbird/cloud/storage/conf/AppConf.scala (hunk 1 above)
Comment (sowmya-dixit) [reply src/main/scala/org/sunbird/cloud/storage/conf/AppConf.scala]: Done - renamed config key

File: src/main/scala/org/sunbird/cloud/storage/conf/AppConf.scala (hunk 2 above)
Comment (sowmya-dixit) [reply src/main/scala/org/sunbird/cloud/storage/conf/AppConf.scala]: Done - renamed as suggested

**Review by maheshkumargangula — APPROVED** [review]

**Pull request merged**